### PR TITLE
Instructions for GCloud bucket upload

### PR DESCRIPTION
**Pull request by ivanzvonkov**
- Rearranged README
- Added instructions for GCloud bucket upload

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 -f https://download.pytorch.org/whl/torch_stable.html
-torch==1.8.0+cpu
-torchvision==0.9.0+cpu
+torch==1.8.0
+torchvision==0.9.0
```
Comment (ivanzvonkov) [inline requirements.txt:3]: It fails with the +cpu tag locally

**Comment by ivanzvonkov** [comment]
@mangomadhava I am going to try to keep these PRs short and sweet, just want your feedback on the changes to the readme [link](https://github.com/nasaharvest/street2sat_website/blob/initial-gcp/README.md) here!

**Comment by mangomadhava** [comment]
Oh looks good :)

**Comment by ivanzvonkov** [comment]
Thanks, I'll merge in next time!

**Pull request merged**